drivers/hmis/app/graphql/mutations/add_household_members_to_enrollment.rb

```diff
@@ -10,8 +10,15 @@ def validate_input(household_id:, start_date:, household_members:)
       errors = HmisErrors::Errors.new
       errors.add :start_date, :out_of_range, message: 'cannot be in the future', readable_attribute: 'Entry date' if Date.parse(start_date) > Date.today
 
-      has_enrollment = Hmis::Hud::Enrollment.editable_by(current_user).exists?(household_id: household_id)
-      has_hoh_enrollment = Hmis::Hud::Enrollment.editable_by(current_user).exists?(household_id: household_id, relationship_to_ho_h: 1)
+      has_enrollment = Hmis::Hud::Enrollment.editable_by(current_user).exists?(
+        household_id: household_id,
+        data_source_id: hmis_user.data_source_id,
+      )
+      has_hoh_enrollment = Hmis::Hud::Enrollment.editable_by(current_user).exists?(
```

Comment:
As an alternate to this (which I think is totally fine) we could add a default scope to the various HUD models and set it to something like `where(data_source_id: hmis_data_source_id)`.  That way we could assume we're always working within the one data source.  We'd probably need to set the `hmis_data_source_id` on boot somehow.

Reply:
It's also possible `editable_by` and `viewable_by` already include the data source id.

Reply:
I thought about that, but the `hmis_data_source_id ` is set per-request based on the origin that the request is coming from. Since we need to support multiple HMIS frontends on the same backend, I think we need to keep that approach.

Reply:
Let's chat about this.  If I'm following, I think it'd make perfect sense to use `hmis_data_source_id` in this scenario.

Reply:
Ok, updated to relied on viewable_by scope as discussed. https://github.com/greenriver/hmis-warehouse/pull/2838/commits/57076f0c03f6ce2a1d1b38670994bebcf55133df